### PR TITLE
Skull token qol

### PR DESCRIPTION
**Pull request by Jagnt**
Auto pickup skulls
Replaced armour pickup sound with chat print for player to notify skull pickup

### CODE REVIEW

File: gamemodes/horde/entities/entities/horde_skull_token/shared.lua [hunk 1]
```diff
@@ -1,2 +1,12 @@
 ENT.Type = "anim"
-ENT.Base = "base_anim"
\ No newline at end of file
+ENT.Base = "base_anim"
+
+if SERVER then
+    util.AddNetworkString("PrintSkullNotify")
```
Comment (wrefgtzweve) [inline gamemodes/horde/entities/entities/horde_skull_token/shared.lua:5]: should be in init.lua

File: gamemodes/horde/entities/entities/horde_skull_token/shared.lua [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+end
+
+if CLIENT then
```
Comment (wrefgtzweve) [inline gamemodes/horde/entities/entities/horde_skull_token/shared.lua:8]: should be in cl_init.lua

Comment (Jagnt) [reply gamemodes/horde/entities/entities/horde_skull_token/shared.lua]: moved both

File: gamemodes/horde/entities/entities/horde_skull_token/cl_init.lua [hunk 3]
```diff
@@ -4,4 +4,10 @@ function ENT:Draw()
     self:DrawModel()
 end
 
-ENT.ColorModulation = Color(1, 0, 1)
\ No newline at end of file
+ENT.ColorModulation = Color(1, 0, 1)
+
+if CLIENT then
```
Comment (wrefgtzweve) [inline gamemodes/horde/entities/entities/horde_skull_token/cl_init.lua:9]: cl_init only runs on client you don't need the CLIENT check, same for SERVER in init.lua

Comment (Jagnt) [reply gamemodes/horde/entities/entities/horde_skull_token/cl_init.lua:9]: oh, k
also gotta love how github doesn't update the conversation tab live and you have to refresh it to see new messages 💀

Comment (Jagnt) [reply gamemodes/horde/entities/entities/horde_skull_token/cl_init.lua]: changed btw

**Review by wrefgtzweve — APPROVED** [review]

**Pull request merged**